src/moin/apps/frontend/views.py

```diff
@@ -572,6 +572,9 @@ def wrapper(item_name, rev):
             abort(404, item_name)
         if add_trail:
             flaskg.user.add_trail(item_name, aliases=item.meta.revision.fqnames)
+        if rev == CURRENT:
+            """if view has been called with default rev=CURRENT we can avoid an index query in flash_if_item_deleted"""
+            item.is_current = CURRENT
```

Comment:
Nitpick:

When using is_something as a name, the name suggests that the value is a boolean, indicating whether the object in question "is something" (or not).

But it looks like the value isn't boolean here.